void tomo::IoLoop::run_loop<false, true, false, false, false>()
--- PRE

+++ POST

@@ -610,7 +610,7 @@

 0b5b [ 5] jmp <+0x4a0>
 0b60 [ 5] mov $0x1,%esi
 0b65 [ 3] mov %rbx,%rdi
-0b68 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0b68 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0b6d [ 5] jmp <+0x2c2>
 0b72 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b7d [ 3] nopl (%rax)
